docs/Readme.md

```diff
@@ -14,6 +14,9 @@ If you would like to see more examples of field data plug-ins, please visit our
 
 ## Change Log
 
+### AQTS 2020.1 - Framework version 2.8
```

Comment:
This is for AQTS 2020.2, not 2020.1, right? There were no framework changes to 2020.1.

Reply:
Ah yeah, for some reason I keep thinking we are on 2020.1, thanks